app/scripts/blocker_tagesschau.js

```diff
@@ -0,0 +1,13 @@
+console.log("#### AfD CONTENT-BLOCKER ####");
+
+import { Blocker } from "./blocker_general";
+
+let blocker = new Blocker([
+    {
+        selector: '.teaser',
+        type: 'big'
+    }
+]);
```

Comment:
I can find some elements that are not blocked by this:
* Article Pages
* TOP 5 lists next to an article